src/montgomery.rs

```diff
@@ -118,6 +118,11 @@ impl Zeroize for MontgomeryPoint {
 }
 
 impl MontgomeryPoint {
+    /// Fixed-base scalar multiplication (i.e. multiplication by the base point).
+    pub fn mul_base(scalar: &Scalar) -> Self {
+        scalar * X25519_BASEPOINT
```

Comment:
Curious if `EdwardsPoint::mul_base(scalar).to_montgomery()` might be faster when `precomputed-tables` is enabled, but that needs a benchmark to evaluate (which might be worth adding to this PR).

That performs another scalar multiplication and an inversion so it's definitely not cheap, but maybe with the basepoint tables it would still be a win.

Reply:
I can benchmark this. Currently x25519-dalek does `EdwardsPoint::mul_base` and converts